smart_contract_examples/SwapExamples.sol

```diff
@@ -126,7 +126,7 @@ contract SwapExamples {
     }
 
     /// @notice swapExactOutputMultihop swaps a minimum possible amount of DAI for a fixed amount of WETH through an intermediary pool.
-    /// For this example, we want to swap DAI for WETH9 through a USDC pool but we specify the desired amountOut of WETH9. Notice how the path encoding is slightly different in for exact output swaps.
+    /// For this example, we want to swap DAI for WETH9 through a USDC pool but we specify the desired amountOut of WETH9. Notice how the path encoding is slightly different in exact output swaps.
```

Comment:
Not sure what `exact output swaps` mean. Can you tell me more hopefully?

Reply:
`exact output swaps` is a type of swap. I think for this we'd like to say something like "Notice how the path encoding is slightly different for exact output swaps." If you make this change I can merge the PR.